src/dosbox.cpp

```diff
@@ -1173,8 +1171,9 @@ void DOSBOX_Init()
 	pstring->Set_help("File used to map fake phone numbers to addresses\n"
 	                  "('phonebook.txt' by default).");
 
-	/* All the DOS Related stuff, which will eventually
-	 * start up in the shell */
+	// All the general DOS Related stuff, on real machines mostly located in
+	// CONFIG.SYS
+
```

Comment:
Thanks for this @FeralChild64 ; this will help us all start to partition settings into this area, especially when there were DOS-equivalents.